packages/logo/README.md

```diff
@@ -0,0 +1,20 @@
+# HeyCar-UIKit - Logo
+
+The Logo component is used to display heycar [logos](https://www.figma.com/file/rrvkvQEoVTOHa7MzyKSaoz/OTP-Global-Design-System?node-id=1370%3A40820).
+
+## Colors
+
+Use the `color` prop to change the visual color of the Logo.
+You can set the value to `primary`, `secondary` , `tertiary`
+
+Mica Blue = Primary
+Heycar Mint = Secondary
+Sunbeam Blue = Tertiary
+
+for more info you checkout colour page in [Figma](https://www.figma.com/file/rrvkvQEoVTOHa7MzyKSaoz/OTP-Design-System?node-id=1291%3A35195)
+
+##installation and usage
+
+import Logo from '@heycar-uikit/logo';
+
+<Logo fontSize={65} color="primary" />
```

Comment:
this section better move to the beginning of the file.
do not forget add link and the storybook.
and wrap please code part of the document to the special code quotes ```` `

Reply:
resolved